app/app.js

```diff
@@ -422,3 +422,21 @@ function ampJsUrl(name, rtv) {
       'http://localhost:8001/dist/v0/' + name + '-0.1.max.js' :
       cdnBase + '/v0/' + name + '-0.1.js';
 }
+
+/**
+ * @param {string} id 
+ */
+function setConfig(id) {
+  const config = getConfig(id);
+  process.env.SERVE_PUBID = config.publicationId;
```

Comment:
We can't be overriding the static shared state with a request state - somewhere it'll really hurt us.

Reply:
what do you suggest we do? This needs to be updated in multiple files right?

Reply:
fixed